docker/planex-container

```diff
@@ -15,6 +15,8 @@ docker pull $PLANEX_CONTAINER
 docker run \
   --privileged \
   --rm -i -t \
+  -e SSH_AUTH_SOCK \
+  -v $(dirname $SSH_AUTH_SOCK):$(dirname $SSH_AUTH_SOCK) \
```

Comment:
Does this work when `SSH_AUTH_SOCK` is empty? `dirname ""` returns `.` which might not be what you want.